src/index.html

```diff
@@ -0,0 +1,253 @@
+<!DOCTYPE html>
+<html lang="pl">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Projekt</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Grand+Hotel&family=Montserrat:wght@400;600&display=swap"
+        rel="stylesheet">
+    <link rel="stylesheet" href="../src/style/style.css">
```

Comment:
Wszystkie ścieżki do plików lokalnych są do poprawy. Powinny być relatywnie względem katalogu **src**, nie powinnaś w ścieżce wychodzić poza ten katalog. Zatem zamiast `../src/style/style.css` powinno być `./style/style.css` i analogicznie we wszystkich ścieżkach.